scripts.py

```diff
@@ -1221,8 +1220,16 @@ def do_run(self):
 
 
 
-class CheckContributorNames(IdentifierInputScript):
-    """ TODO """
+class CheckContributorNamesInDB(IdentifierInputScript):
+    """ Checks that contributor sort_names are display_names in 
+    "last name, comma, other names" format.  
+
+    NOTE:  There's also CheckContributorNamesOnWeb in metadata, 
+    it's a child of this script.  Use it to check our knowledge against 
+    viaf, with the newer better sort_name selection and formatting.
+
+    TODO: make sure don't start at beginning again when interrupt while batch job is running.
```

Comment:
I don't know.  We'll only run it once, hopefully, but the metadata database is very large, and things can crash.  Won't hurt to try and see.